x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_content/log_rate_analysis_content.tsx

```diff
@@ -64,6 +65,8 @@ export interface LogRateAnalysisContentProps {
   barHighlightColorOverride?: string;
   /** Optional callback that exposes data of the completed analysis */
   onAnalysisCompleted?: (d: LogRateAnalysisResultsData) => void;
+  /** Optional identifier to indicate the plugin utilizing the component */
+  source?: string;
```

Comment:
I wonder if we should make this non-optional to enforce correct telemetry on new/future embeddings?

Reply:
Good call - updated in bc27b463adabda26003a2e8fc3bbc5b1cf796a62